### PR TITLE
Docs - Develop - Core Concepts - Core Contracts 

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

A few small edits to the Core Concepts > Core Contracts.  Mostly around using consistent formatting. 

## Type of change

- Documentation enhancement

## How the change has been tested

Wiki was built locally

## Change checklist

- [x] I have followed the [contribution guidelines](https://wiki.iota.org/smart-contracts/contribute) for this project
- [x] I have selected the `develop` branch as the target branch
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by dessaya — APPROVED** [review]

**Comment by lucas-tortora** [comment]
@iotaledger/tech-writers , whenever you have a minute, could you please take a look at this?

File: documentation/docs/guide/core_concepts/core_contracts/accounts.md [hunk 1]
```diff
@@ -109,84 +122,90 @@ Parameters:
...
+#### Parameters
 
-- `a` (`AgentID`): The account Agent ID
+- `a` (`AgentID`): The account Agent ID.
 
-Returns:
+#### Returns
 
-A map of [`TokenID`](#tokenid) => `big.Int`. The L1 base tokens is represented by an empty Token ID (a key with length 0).
+A map of [`TokenID`](#tokenid) => `big.Int`. An empty Token ID represents the L1 base tokens (a key with length 0).
```
Comment (aleksei-korolev) [inline documentation/docs/guide/core_concepts/core_contracts/accounts.md]:
```suggestion
A map of [`TokenID`](#tokenid) => `big.Int`. An empty token ID (a string of zero length) represents the L1 base token.
```

File: documentation/docs/guide/core_concepts/core_contracts/accounts.md [hunk 2]
```diff
@@ -109,84 +122,90 @@ Parameters:
...
 
 ### `totalAssets()`
 
 Returns the sum of all fungible tokens controlled by the chain.
 
-Returns:
+#### Returns
 
-A map of [`TokenID`](#tokenid) => `big.Int`. The L1 base tokens is represented by an empty Token ID (a key with length 0).
+A map of [`TokenID`](#tokenid) => `big.Int`. An empty Token ID represents the L1 base tokens (a key with length 0).
```
Comment (aleksei-korolev) [inline documentation/docs/guide/core_concepts/core_contracts/accounts.md]:
```suggestion
A map of [`TokenID`](#tokenid) => `big.Int`. An empty token ID (a string of zero length) represents the L1 base token.
```

File: documentation/docs/guide/core_concepts/core_contracts/blob.md [hunk 3]
```diff
@@ -27,20 +27,20 @@ A blob is a collection of named chunks of binary data.
 
 Both names and chunks are arbitrarily long byte slices.
 
-Blobs can be used to store arbitrary data; for example, the collection of Wasm binaries needed to deploy a smart contract.
+Blobs can be used to store arbitrary data. For example, the collection of Wasm binaries needed to deploy a smart contract.
```
Comment (aleksei-korolev) [inline documentation/docs/guide/core_concepts/core_contracts/blob.md]:
```suggestion
Blobs can be used to store arbitrary data like, for example, a collection of Wasm binaries needed to deploy a smart contract.
```

Is it possible to be more specific? What exactly uses these binaries and what exactly for? For example, "...utility binaries that are referenced by all smart contracts"

File: documentation/docs/guide/core_concepts/core_contracts/blocklog.md [hunk 4]
```diff
@@ -148,35 +153,35 @@ Returns:
...
-- Total storage deposit (`uint64`)
-- Gas burned (`uint64`)
-- Gas fee charged (`uint64`)
+- The block timestamp (`uint64` UNIX nanoseconds).
+- Amount of requests in the block (`uint16`).
+- Amount of successful requests (`uint16`).
+- Amount of off-ledger requests (`uint16`).
+- Anchor transaction ID ([`iotago::TransactionID`](https://github.com/iotaledger/iota.go/blob/develop/transaction.go)).
+- Anchor transaction sub-essence hash (`[32]byte`).
+- Previous L1 commitment (except for block index 0.)
```
Comment (aleksei-korolev) [inline documentation/docs/guide/core_concepts/core_contracts/blocklog.md]:
```suggestion
- Previous L1 commitment (except for block index 0).
```

One is not like the others.

File: documentation/docs/guide/core_concepts/core_contracts/errors.md [hunk 5]
```diff
@@ -60,13 +66,16 @@ Returns:
...
+- The error ID, calculated as the hash of the error template(`uint16`).
 
 ### `UnresolvedVMError`
 
 `UnresolvedVMError` is encoded as the concatenation of:
 
-- ([`ErrorCode`](#errorcode)) The error code
-- (`uint32`) CRC32 checksum of the formatted string
-- (`string` prefixed with `uint16` size) The JSON-encoded list of parameters for the template
+- The error code ([`ErrorCode`](#errorcode)) .
```
Comment (aleksei-korolev) [inline documentation/docs/guide/core_concepts/core_contracts/errors.md]:
```suggestion
- The error code ([`ErrorCode`](#errorcode)).
```

File: documentation/docs/guide/core_concepts/core_contracts/evm.md [hunk 6]
```diff
@@ -27,34 +31,39 @@ For more information about how ISC supports EVM contracts, refer to the [EVM](..
...
-  Re-using an existing Chain ID is not recommended, and can be a security risk. For any serious usage make sure you register a unique Chain ID on [Chainlist](https://chainlist.org/) and use that instead of the default. **It is not possible to change the EVM chain ID after deployment.**
+
+  Re-using an existing Chain ID is not recommended and can be a security risk. For serious usage, register a unique
+  Chain ID on [Chainlist](https://chainlist.org/) and use that instead of the default. **It is not possible to change
+  the EVM chain ID after deployment.**
+
   :::
 
-- `evmw` (optional [`GasRatio`](#gasratio) - default: `1:1`) The ISC : EVM gas ratio.
+- `evmw` (optional [`GasRatio`](#gasratio) - default: `1:1`): The ISC : EVM gas ratio.
```
Comment (aleksei-korolev) [inline documentation/docs/guide/core_concepts/core_contracts/evm.md]:
```suggestion
- `evmw` (optional [`GasRatio`](#gasratio) - default: `1:1`): The ISC to EVM gas ratio.
```

**Review by aleksei-korolev — CHANGES_REQUESTED** [review]

File: documentation/docs/guide/core_concepts/core_contracts/blob.md (hunk 3 above)
Comment (lucas-tortora) [reply documentation/docs/guide/core_concepts/core_contracts/blob.md]: I'm afraid I don't have the answer to your question

**Review by aleksei-korolev — APPROVED** [review]

**Pull request merged**